workshop/templates/workshop/workshop_list.html

```diff
@@ -24,6 +24,9 @@ <h1 class="hero-title font-alt">Workshops</h1>
 
             <div style ="padding :30px " class="card-block">
                 <h1 class="card-title">Workshops</h1>
+                <a href="{% url 'workshop_create' %}">
```

Comment:
This button should be visible only for admins. Now all users and even the public can see this.